examples/calender/backend/models/appointment.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Model } from "sequelize";
+
+export class Appointment extends Model {
+  public id!: number; // Note that the `null assertion` `!` is required in strict mode.
+  public fromUserId!: string;
```

Comment:
`fromUser` and `toUser`
I would suggest to use lowerCamelCase for timeStart and timeEnd